### PR TITLE
[ROCKETMQ-220]Add IT test for Filter By SQL 92

### PR DESCRIPTION
**Pull request by dongeforever**
https://issues.apache.org/jira/browse/ROCKETMQ-220

@lizhanhui @zhouxinyu

### CODE REVIEW

File: test/src/main/java/org/apache/rocketmq/test/listener/AbstractListener.java [hunk 1]
```diff
@@ -95,6 +95,28 @@ public void stopRecv() {
...
+    public long waitForMessageConsume(int size,
+        int timeoutMills) {
+
+        long curTime = System.currentTimeMillis();
+        while (true) {
+            if (msgBodys.getDataSize() >= size) {
+                break;
+            }
+            if (System.currentTimeMillis() - curTime >= timeoutMills) {
+                logger.error(String.format("timeout but  [%s]  not recv all send messages!",
```
Comment (lizhanhui) [inline test/src/main/java/org/apache/rocketmq/test/listener/AbstractListener.java]: but  [%s]  not recv all send messages! --> but [%] did not receive all messages sent!

**Comment by lizhanhui** [comment]
+1

**Comment by zhouxinyu** [comment]
+1, LGTM~

**Comment by vongosling** [comment]
+1

**Comment by dongeforever** [comment]
@zhouxinyu Thanks

**Pull request closed without merging**